test-harness/modules/mocker/endpoints/marathon.py

```diff
@@ -409,3 +423,9 @@ def break_leader_reply(self, *_):
         """
         with self._context.lock:
             self._context.data["leader-content"] = 'blah blah buh buh'
+
+    def _reset(self):
```

Comment:
I'd rather do it in separate story, this is just a patch for not having duplicate code in a single class.